generator/ServiceClientGeneratorLib/ServiceClientGeneratorLib.csproj

```diff
@@ -735,9 +735,12 @@
       <LastGenOutput>AmazonS3RetryPolicy.cs</LastGenOutput>
     </None>
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
 	<PackageReference Include="Microsoft.CSharp" Version="4.4.1" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net45'">
+	<Reference Include="Microsoft.CSharp" />
```

Comment:
did we want to make this use the same version as above (4.4.1)?

Reply:
The reference is provided by the framework, not a Nuget package.